guides/03_API_Overview.md

```diff
@@ -174,6 +174,11 @@ To show speaker volume, mute state, and signal strength for each attendee, subsc
 
 Volume is on a scale of 0 to 1 (no volume to max volume). Signal strength is on a scale of 0 to 1 (full packet loss to no packet loss). You can use the signal strength of remote attendees to show an indication of whether an attendee is experiencing packet loss and thus may be unable to communicate at the moment.
 
+Signal strength
+* 0 - 100% loss of audio data
```

Comment:
Could we keep this in sync with  https://aws.github.io/amazon-chime-sdk-js/modules/qualitybandwidth_connectivity.html#events-for-monitoring-remote-attendee-uplink. So its fractional in any range and not specific values only?

Reply:
yes, this change is in sync with qualitybandwidth_connectivity descriptions. The signal strength values only have 0, 05. and 1.